package.json

```diff
@@ -161,7 +161,7 @@
     "textarea-caret": "^3.0.1",
     "valid-url": "^1.0.9",
     "web3": "^0.20.7",
-    "web3-stream-provider": "^3.0.1",
+    "web3-stream-provider": "rekmarks/web3-stream-provider#id-remap",
```

Comment:
We should upstream this so we're not using a Git URL with a branch reference

Reply:
(As shown below.)

Reply:
Pending: https://github.com/kumavis/web3-stream-provider/pulls

Reply:
published as `web3-stream-provider@4.0.0`

Reply:
Package updated.